### PR TITLE
Updated the PKA.jsx to send set Algorithm

### PR DESCRIPTION
**Pull request by iniyanr**

### CODE REVIEW

**Review by carb1d3 — CHANGES_REQUESTED** [review]
Remove the print statement line 24 - 26

File: src-tauri/src/main.rs [hunk 1]
```diff
@@ -25,6 +21,9 @@ fn password_auth(username: &str) {
 
 #[tauri::command]
 fn generate_keys(algorithm: &str, password: &str) {
+    // Print the algorithm value received
```
Comment (carb1d3) [inline src-tauri/src/main.rs]: Remove this change

File: src/pages/PKA.jsx [hunk 2]
```diff
@@ -55,7 +55,7 @@ const PKA = () => {
         </Button>
         <p className="text-xl text-white">
           Copy the key {algorithm}.pub file generated in <code>~/.ssh/ </code>{" "}
-          to the host systems <code> ~/.ssh/authorized_keys </code> file.
```
Comment (carb1d3) [inline src/pages/PKA.jsx:58]: Keep all other changes except line 24 - 26

**Review by carb1d3 — CHANGES_REQUESTED** [review]

File: src/pages/PKA.jsx (hunk 2 above)
Comment (iniyanr) [reply src/pages/PKA.jsx:58]: Done

**Review by carb1d3 — APPROVED** [review]
👍🏼

**Pull request merged**